app/components/SplashScreen.tsx

```diff
@@ -37,7 +37,7 @@ const SplashScreen: FC = () => {
   // scales (either 'size' or 'height'; height may be better for estimating)
   // in-line styling
   return (
-    <Box className={classes.root}>
+    <Box className={classes.root} data-testid="splash-screen">
```

Comment:
Nit, non-blocker. I've been using the pattern of naming the ID to match the component.

At the top level, this would be `SplashScreen`
At the LogoIcon, it would be `SplashScreen__LogoIcon`
And, for example, a cool button in the component may be `ComponentName__cool-button`

Something like a modified BEM.

https://en.bem.info/methodology/naming-convention/